Description: Fix array initialization bug
Author: Phil Nelson
Origin: upstream
Bug-Debian: http://bugs.debian.org/586969
Bug-Debian: http://bugs.debian.org/671513
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: bc/bc/storage.c
===================================================================
--- bc.orig/bc/storage.c	2013-06-02 20:34:09.401273864 -0400
+++ bc/bc/storage.c	2013-06-02 20:34:11.000000000 -0400
@@ -179,7 +179,7 @@
 
 
   /* Initialize the new elements. */
-  for (; indx < v_count; indx++)
+  for (; indx < a_count; indx++)
     arrays[indx] = NULL;
 
   /* Free the old elements. */
